libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/FacebookAdapter.cs

```diff
@@ -51,19 +56,19 @@ public FacebookAdapter(FacebookClientWrapper facebookClient)
         /// <summary>
         /// Standard BotBuilder adapter method to send a message from the bot to the messaging API.
         /// </summary>
-        /// <param name="context">A TurnContext representing the current incoming message and environment.</param>
+        /// <param name="turnContext">A TurnContext representing the current incoming message and environment.</param>
         /// <param name="activities">An array of outgoing activities to be sent back to the messaging API.</param>
         /// <param name="cancellationToken">A cancellation token for the task.</param>
         /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
-        public override async Task<ResourceResponse[]> SendActivitiesAsync(ITurnContext context, Activity[] activities, CancellationToken cancellationToken)
+        public override async Task<ResourceResponse[]> SendActivitiesAsync(ITurnContext turnContext, Activity[] activities, CancellationToken cancellationToken)
         {
             var responses = new List<ResourceResponse>();
 
             foreach (var activity in activities)
             {
-                if (activity.Type != ActivityTypes.Message)
+                if (activity.Type != ActivityTypes.Message && activity.Type != ActivityTypes.Event)
                 {
-                    throw new Exception("Only Activities of type Message are supported for sending.");
+                    continue;
```

Comment:
What is the point of this? Maybe this if could be removed.

Reply:
It's there to mirror a similar evaluation in the Node JS version to prevent processing activities that don't match with any of those two types. I don't know, maybe it could be removed. Not sure.

Reply:
But with the `continue` you're not preventing the process.